test/loadmodels.jl

```diff
@@ -214,7 +235,7 @@ end
     @test test_math.args[2].fn == "sin"
     @test test_math.args[2].args[1].val == 2.1
 
-    @test_logs (:warn,) (:warn,) (:warn,) (:warn,) readSBML(
+    @test_logs (:warn,) (:warn,) (:warn,) (:warn,) (:warn,) readSBML(
```

Comment:
This is allowing an extra warning about an unsupported math element so that tests will pass for the time being, but we may want to fix that before merging this PR